From 344a8f57c6417706dd88d4311dc9f54057900cee Mon Sep 17 00:00:00 2001
From: Shenwei Wang <shenwei.wang@nxp.com>
Date: Mon, 7 May 2018 09:51:33 -0500
Subject: [PATCH 3715/5242] MLK-18219: arm: dts: add PMU node for 7ULP boards

commit  3c32045351b688ca4c545fdbdfe33943b089f9ee from
https://source.codeaurora.org/external/imx/linux-imx.git

Added the PMU node in the imx7ulp.dtsi, and enable
it by default.

Signed-off-by: Shenwei Wang <shenwei.wang@nxp.com>
Reviewed-by: Frank Li <frank.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index fd5061c..19c0d86 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -142,6 +142,10 @@
 		reg = <0x1fffc000 0x4000>;
 	};
 
+	pmu: pmu@a7 {
+		compatible = "arm,cortex-a7-pmu";
+	};
+
 	ahbbridge0: ahb-bridge0@40000000 {
 		compatible = "fsl,aips-bus", "simple-bus";
 		#address-cells = <1>;
-- 
1.7.9.5

